docker-stack.yml

```diff
@@ -87,10 +102,9 @@ services:
 
                 if [ $$django_tables_exist -gt 0 ] || [ $$REFRESH_DB == 'true' ]; then
                     ./refresh-data.sh 
+                    ./cfgov/manage.py update_index
                 fi
 
-                ./cfgov/manage.py update_index
-                
```

Comment:
Are you intentionally reverting the fix that Marc made in #6045? As noted there, the current situation is not ideal, but better than it was, I think.

Reply:
Nope - will add it back into the mix.